chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/CertifiedProductController.java

```diff
@@ -109,12 +110,12 @@
     @Autowired
     private CertifiedProductManager cpManager;
 
-    @Autowired 
+    @Autowired
```

Comment:
Could we start moving  our field injections to constructor injections so we are following Spring's preferred method?

Reply:
I can update the existing clas sure